conf/log4j.properties

```diff
@@ -1,7 +1,7 @@
 hcfsfuse.logger.type=TEST_LOGGER
-hcfsfuse.root.logger=INFO, ${hcfsfuse.logger.type}
+hcfsfuse.root.logger=${hcfsfuse.logger.type}
 
-log4j.rootLogger=${hcfsfuse.root.logger},stdout
+log4j.rootLogger=WARN, ${hcfsfuse.root.logger},CONSOLE
```

Comment:
Default to INFO might be better? Error>Warn>Info>Debug

Reply:
@Xushaohong Thanks for this suggestion.